process/factory/shard/interceptorsContainerFactory.go

```diff
@@ -483,7 +483,8 @@ func (icf *interceptorsContainerFactory) generateHdrInterceptor() ([]string, []p
 		return nil, nil, err
 	}
 
-	identifierHdr := factory.HeadersTopic + shardC.CommunicationIdentifier(shardC.SelfId())
+	// shardBlocks_0_META, shardBlocks_1_META....
```

Comment:
no need for comment, and you have only one identifier, comment is misleading.

Reply:
added for example